bsk_rl/envs/general_satellite_tasking/simulation/dynamics.py

```diff
@@ -841,12 +841,15 @@ def _set_instrument(
             self.task_name, self.instrument, ModelPriority=priority
         )
 
-    @default_args(dataStorageCapacity=20 * 8e6, storageUnitValidCheck=True)
+    @default_args(
+        dataStorageCapacity=20 * 8e6, storageUnitValidCheck=True, setStorageInit=0
+    )
     def _set_storage_unit(
         self,
         dataStorageCapacity: int,
         priority: int = 699,
         storageUnitValidCheck: bool = True,
```

Comment:
Does this still work how we wanted with the change to integers? I think so, but not sure if the floating point error was part of that

Reply:
Good point. The storage is not supposed to go over its limit anymore due to changes in the baseStorageUnit module (using integers and how it checks for free space). But storageValidCheck will still check if the storage unit is at its maximum capacity. So, I believe it still has the expected behavior.